syncd/Syncd.cpp

```diff
@@ -961,6 +960,14 @@ sai_status_t Syncd::processBulkQuadEventInInitViewMode(
                     return SAI_STATUS_SUCCESS;
             }
 
+        case SAI_COMMON_API_BULK_SET:
```

Comment:
description says hat you support PORT and IPG, but you also supported here SWITCH and SCHEDULER, code is not duing what's expected. please remove at least SWITCH support.

PS. why do you need SET support on init view on those objects ? when initview is compiled, all objects should already be known and its attributes so no SET actions should be added to those excluded objects

Reply:
also if you are changing this, please add recodring TESTS in initview if this scenario will pass

Reply:
@kcudnik Changed the title. Got this error when configuring PGs in warm start in orchagent on VS:
```
ERR #syncd: :- run: Runtime error: :- processBulkQuadEventInInitViewMode: SAI_OBJECT_TYPE_INGRESS_PRIORITY_GROUP is not supported in init view mode
```

I don't see any restrictions on non-bulk set in init view, neither for PGs, ports nor switch.
Unit tests are present and passes coverage requirement.

Reply:
i dont remember why i put those restrictions on the first place, dont remember exactly now, but they were there for a reason, SWITCH should be remove from this SET, since if you are doing CREATE_SWITCH then all attributes should be posted already to that create function, you can change other later on. switch is special case, since for example you can set some ACL etc, and those objects may not exists before you create switch

Reply:
@kcudnik ok, added exception for switch and scheduler group back